### PR TITLE
Added user-stories to the read me.

### PR DESCRIPTION
**Pull request by Amirk390**
Added user-stories to the read me. 
Reference #13 #18

### CODE REVIEW

**Review by SamiSha99 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
 ## Team:
 
 Developers: @Amirk390 and @SamiSha99
+
 Scrum: @Karyum
+
```
Comment (matthewdking) [inline README.md]: Scrum **master**

File: README.md [hunk 2]
```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
...
 
 
 1. clone it `git clone git@github.com:FACN3/washinator.git` in your tarminal
-+ then `npm i`
+2. install npm packages  `npm i`
 
 See also the list of [contributors](https://github.com/FACN3/washinator/graphs/contributors) who participated in this project.
 
-## Our ProtoType
+# Our ProtoType
```
Comment (matthewdking) [inline README.md]: Prototype shouldn't be camelCase

File: README.md [hunk 3]
```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
...
+Below, we have some user storys and the solutions for that storys
```
Comment (matthewdking) [inline README.md]: *stories

File: README.md [hunk 4]
```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
...
 
-[Our InVision ProtoType](https://invis.io/UNFO2C1YA2P)
 
-[Download link](https://projects.invisionapp.com/d/zipexport/download/projectid/13267105)
+### User Story:
```
Comment (matthewdking) [inline README.md]: I think I would move this section further up the README

File: README.md [hunk 5]
```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
...
+:man: As a water consumer, I want, to calculate my overall water consumption.
+So that i can reduce my water expenses
```
Comment (matthewdking) [inline README.md]: As a water consumer, I want, to calculate my overall water consumption, so that i can reduce my water expenses.

All one sentence :point_up:

File: README.md [hunk 6]
```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
...
 
 ### Calculator
 
 ![Calculator](https://user-images.githubusercontent.com/24490876/35803983-846a0468-0a7e-11e8-9b37-d00265775367.png)
 
+### User Story:
+:raising_hand: As a Water consumer, I want to choose a certain activity and know how much water being used also how much money dollar So that I would be more careful with using Water.
```
Comment (matthewdking) [inline README.md]: As a water consumer, I want to choose certain activities and know how much water I am using and how much money this equates to, so that I will be more careful with the amount of water I use.

File: README.md [hunk 7]
```diff
@@ -8,36 +8,53 @@ Our website focus on reducing water consumption and helping people calculate how
...
+
+
+
 ### Specific Activity Prices
 
 ![Prices](https://user-images.githubusercontent.com/24490876/35803984-848d2ea2-0a7e-11e8-8962-71bc4fea2874.png)
+
+## Prototype links
+
```
Comment (matthewdking) [inline README.md]: These links should just go under the Prototype title :point_up:

**Review by matthewdking — CHANGES_REQUESTED** [review]
Few small grammar changes :smile:

**Review by matthewdking — COMMENTED** [review]
Also your PR should always have a description.

File: basic-counter/app.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
+class App extends React.Component {
```
Comment (matthewdking) [inline basic-counter/app.js]: All these files :point_down: shouldn't be included in this PR

Double check what you are commiting by using `git status` before you commit.

**Review by matthewdking — CHANGES_REQUESTED** [review]

File: basic-counter/app.js (hunk 8 above)
Comment (Amirk390) [reply basic-counter/app.js]: ops

File: LICENSE [hunk 9]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```
Comment (matthewdking) [inline LICENSE:1]: Did you mean to delete the license? :man_shrugging:

Comment (Amirk390) [reply LICENSE:1]: @matthewdking Yeah, i didn't see the point of having it at the moment :sweat_smile:

**Review by matthewdking — APPROVED** [review]
